project/Dependencies.scala

```diff
@@ -3,8 +3,7 @@ import sbt._
 object Dependencies {
   val akkaV = "2.4.19"
   val sprayV = "1.3.4"
-  val jacksonV = "2.8.10"
-     // note that jackson-databind overrides this below! 2.8.8.1 is not released for core or annotations.
```

Comment:
Is this still true?

Reply:
it is not true for 2.9.6 - annotatinos, databind, and core are all available at 2.9.6. In fact, this comment became untrue as of #570 but I failed to remove it then.